util/travis/script.sh

```diff
@@ -21,8 +26,17 @@ else
 	# \/  \/  \/
 	# /home/travis/minetest/minetest/travisbuild/minetest/travisbuild/minetest/travisbuild/minetest
 	# You get the idea.
-	OLDDIR=`pwd`
+	OLDDIR=$(pwd)
```

Comment:
Backtick format is deprecated because it's harder to read, especially when it's nested (although it's technically still supported).